roles/gateway/templates/auth-openvpn

```diff
@@ -36,16 +36,16 @@ def authenticate():
     #
     # We'll just redo that quickly for usernames, to ensure they are safe.
 
-    user = re.sub(r'[^a-zA-Z0-9_.@-]', '_', user)
+    user = re.sub(rb'[^a-zA-Z0-9_.@-]', '_', user)
 
     def find_user(wanted):
-        with file('{{ openvpn_data_dir }}/users') as f:
+        with open('/etc/openvpn/cephco-sepia.data/users', 'rb') as f:
```

Comment:
Why drop the templating here? Does the role need adjustment elsewhere to complement this change?

Reply:
Mistake.  I was applying changes I'd actually tested by modifying the source, and didn't notice the change of template; was focused on the 'rb'.  fixing.

Reply:
forcepushed a fix for the dropped template.